config/example_config.yml

```diff
@@ -37,6 +37,6 @@ shared:
   # What version of kubernetes will be deployed?
   # https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-6-9/
   rancher_on_eks:
-    k8s_version: 1.23
+    k8s_version: "1.23"
   rancher_on_aks:
-    k8s_version: 1.24.3
+    k8s_version: "1.24"
```

Comment:
are we not converting to string in `engines/azure/app/models/azure/kubernetes_info.rb` line 5 ?

Reply:
belts & suspenders - because the config didn't explicitly have string before now.